src/scales/quantitativeScale.ts

```diff
@@ -152,7 +153,28 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     });
     var newMin = minExistsInExceptions ? min : this.invert(this.scale(min) - (this.scale(max) - this.scale(min)) * p);
     var newMax = maxExistsInExceptions ? max : this.invert(this.scale(max) + (this.scale(max) - this.scale(min)) * p);
-    return this._niceDomain([newMin, newMax]);
+    if (this._niceDomainEnabled) {
+      return this._niceDomain([newMin, newMax]);
+    }
+    return ([newMin, newMax]);
+  }
+
+  /**
+   * Gets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(): boolean;
+  /**
+   * Sets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(niceDomainEnabled: boolean): QuantitativeScale<D>;
```

Comment:
I think this will be solved by renaming the method. The convention is also to assign the parameter name to the instance variable with the same name, but in our case _niceDomain() is taken